modules/Naive UI.yml

```diff
@@ -0,0 +1,16 @@
+name: Naive UI
+description: Nuxt 3 Naive UI library
```

Comment:
Do not advertize with a specific Nuxt version, see here:

https://nuxt.com/docs/guide/going-further/modules#do-not-advertize-with-a-specific-nuxt-version